packages/cactus-plugin-ledger-connector-ethereum/src/test/typescript/integration/geth-contract-deploy-and-invoke-using-json-object-v1.test.ts

```diff
@@ -423,12 +423,10 @@ describe("Ethereum contract deploy and invoke using keychain tests", () => {
           secret: "",
           type: Web3SigningCredentialType.GethKeychainPassword,
         },
-      } as InvokeContractV1Request);
-      fail(
-        "Expected deployContractSolBytecodeV1 call to fail but it succeeded.",
-      );
-    } catch (error) {
-      console.log("deployContractSolBytecodeV1 failed as expected");
-    }
+      } as InvokeContractV1Request)
+    ).rejects.toThrow();
+    
+    console.log("deployContractSolBytecodeV1 failed as expected");
```

Comment:
@ashnashahgrover This line can be deleted

Reply:
Instantiating a logger object inside and replaced all the console.log statements with the object